### PR TITLE
Changes to the OpenVINO 2.0 Transition Guide

### PR DESCRIPTION
**Pull request by andrew-zaytsev**
### Details:
 Minor changes to the OpenVINO 2.0 Transition Guide

### Tickets:
 CVS-80835

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
Let's merge https://github.com/openvinotoolkit/openvino/pull/10933 first

**Comment by ilya-lavrenov** [comment]
@andrew-zaytsev please, see the latest version and check whether some changes are still needed.

File: docs/OV_Runtime_UG/migration_ov_2_0/intro.md [hunk 1]
```diff
@@ -11,66 +11,68 @@
...
 - Users need to specify input shapes during model conversions in Model Optimizer and work with static shapes in the application.
 
-OpenVINO API 2.0 is introduced to align logic of working with model as it is done in the frameworks - no layout and precision changes, operates with tensor names and indeces to address inputs and outputs. OpenVINO Runtime is composed of Inference Engine API used for inference and ngraph API targeted to work with models, operations. The API 2.0 has common structure, naming convention styles, namespaces, removes duplicated structures. See [How to migrate to API 2.0](./common_inference_pipeline.md) for details.
+OpenVINO™ introduces API 2.0 to align logic of working with model as it is done in the frameworks - no layout and precision changes, operates with tensor names and indices to address inputs and outputs. OpenVINO Runtime is composed of Inference Engine API used for inference and nGraph API targeted to work with models, operations. The API 2.0 has common structure, naming convention styles, namespaces, removes duplicated structures. See [How to migrate to OpenVINO API v2](common_inference_pipeline.md) for details.
```
Comment (ilya-lavrenov) [inline docs/OV_Runtime_UG/migration_ov_2_0/intro.md:27]: why you changed to v2?? In other places of the document we have 2.0

CC @xu-yuan1 what is the proper name?

Comment (xu-yuan1) [reply docs/OV_Runtime_UG/migration_ov_2_0/intro.md:27]: The approved names by marketing are "API 2.0", "OpenVINO API v2" and "OpenVINO 2022.1". I think "OpenVINO API 2.0" is in a grey area as it's not forbidden explicitly. @andrew-zaytsev  Andrey may know better and confirm.
But I think on the same page we should keep consistent, having both "2.0" and "v2" can be confusing. Or maybe we can have a statement at the beginning to say that they are the same thing.

**Comment by ilya-lavrenov** [comment]
@andrew-zaytsev please, port these changes to master if needed

**Pull request merged**